models/lobby.model.js

```diff
@@ -25,19 +69,73 @@ const lobbySchema = new mongoose.Schema({
         type: Schema.Types.ObjectId,
         ref: 'User',
     }],
-    round: {
-        type: Schema.Types.ObjectId,
-        ref: 'Round',
-    },
     lobbyLocked: {
         type: Boolean,
         default: false,
     },
-    rounds: {
-        type: [Number],
-        enum: [1, 2, 3],
-        default: [1],
-    },
+    rounds: [{
+        round: {
+            type: String,
+            enum: roundTypes,
+            default: '1',
+        },
+        loop1: {
```

Comment:
What does `loop1` `loop2` represent? Also, this is not appropriate naming for db schema. We should change it to `firstLoop`, `secondLoop`